configure/README.md

```diff
@@ -0,0 +1,45 @@
+# Telepresence configure action
+
+The Telepresence configure is a one-time per repository action that installs the binary and creates a cache entry with the contents of the Telepresence configuration directory which includes the Telepresence ID obtained after connecting and the user's configuration file (optional).
```

Comment:
Are we sure it is just one-time action? I think we can use it in other scenarios right? Like when updating the configuration or our TP version?

Reply:
Yes, it isn't a one-time action because we can change the config file anytime